source/_integrations/shelly.markdown

```diff
@@ -43,34 +43,40 @@ Integrate [Shelly devices](https://shelly.cloud) into Home Assistant.
 
 ## Shelly device generations
 
-There are two generations of devices. Both generations are supported by this integration. There are some differences in how devices should be configured and in the naming of entities and devices between generations.
+There are two generations of Shelly devices, and this integration supports both. In Home Assistant UI, on each Shelly device's page, **Device info** section will state `Hardware: gen1` or `Hardware: gen2`. There are differences in how generation 1 vs. generation 2 devices should be configured, and in the naming of entities and devices.
 
-## Shelly device configuration (generation 1)
+<div class="note">Shelly cloud connection is not required for this integration to work. Shelly devices are communicating directly with Home Assistant server.</div>
 
-Generation 1 devices use the `CoIoT` protocol to communicate with the integration. For Shelly firmware 1.10.0 or newer, `CoIoT` must be enabled in the device settings. Navigate to the local IP address of your Shelly device, **Internet & Security** >> **ADVANCED - DEVELOPER SETTINGS** and check the box **Enable CoIoT**.
 
-We recommend using `unicast` for communication. To enable this, enter the local IP address of the Home Assistant server and port `5683` into the **CoIoT peer** field and push **SAVE** button. **This is mandatory for battery operated devices**. After changing the **CoIoT peer**, the Shelly device needs to be manually restarted.
+## Configuring 1st generaton Shelly devices
 
-Home Assistant will display a repair issue for the Shelly device if push updates from this device do not reach the Home Assistant server.
+First generation Shelly devices use the `CoIoT` protocol to communicate with the integration. Home Assistant will display a repair issue for any such Shelly device, if push updates from it do not reach the Home Assistant server. To fix:
 
-The list below will help you diagnose and fix the problem:
+1. Ensure the Shelly device is on the current firmware. Older firmware will lack the necessary setting;
```

Comment:
Good idea & intention - I'm not aware if ALL devices share the same firmware version # nomenclature. The goal of this PR is to provide actionable helpful steps to as many people with this issue as possible, as soon as possible - also heeding the Home Assistant dev docs advice to keep PRs small and fast rather than make them the be-all end-all.